### PR TITLE
fix: alert follows fiori3

### PR DESCRIPTION
**Pull request by JKMarkowski**
## Related Issue
Closes SAP/fundamental-styles#452

## Description
There is corrected styling for alert component. Now it follows latest fiori3 requirements.

### CODE REVIEW

**Comment by estelaxu3** [comment]
The update matches with the specs.

**Comment by JKMarkowski** [comment]
Everything is fixed now, Icons are not aligned well, when `fd-link` is being used. It will be fixed automatically after merging https://github.com/SAP/fundamental-styles/pull/475

File: src/alert.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@ $block: #{$fd-namespace}-alert;
 .#{$block} {
   $fd-alert-border: 1px solid;
   $fd-alert-border-radius: 0.25rem;
-  $fd-alert-padding: 0.5rem 1rem;
+  $fd-alert-padding: 0.375rem 1rem;
```
Comment (InnaAtanasova) [inline src/alert.scss]: I believe the top and bottom padding should be 0.4375rem. In specs they say 0.5rem (including border)

File: src/alert.scss [hunk 2]
```diff
@@ -21,16 +21,19 @@ $block: #{$fd-namespace}-alert;
...
-    width: 2rem;
-    height: 1.625rem;
-    top: 0.375rem;
+    top: 0.125rem;
     right: 0.125rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-width: 2rem;
+    height: 1.625rem;
```
Comment (InnaAtanasova) [inline src/alert.scss]: The interaction states of the button should follow these of the Transparent button

**Comment by droshev** [comment]
@JKMarkowski the pipeline is failing

**Pull request merged**